packages/angular/angular.json

```diff
@@ -1,68 +1,84 @@
-					}
-				}
-			}
-		},
-		"charts-angular-e2e": {
-			"root": "e2e",
-			"sourceRoot": "e2e",
-			"projectType": "application",
-			"architect": {
-				"e2e": {
-					"builder": "@angular-devkit/build-angular:protractor",
-					"options": {
-						"protractorConfig": "./protractor.conf.js",
-						"devServerTarget": "charts-angular:serve"
-					}
-				},
-				"lint": {
-					"builder": "@angular-devkit/build-angular:tslint",
-					"options": {
-						"tsConfig": ["e2e/tsconfig.e2e.json"],
-						"exclude": ["./node_modules", "../../node_modules"]
-					}
-				}
-			}
-		}
-	},
-	"defaultProject": "charts-angular",
-	"schematics": {
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "version": 1,
+  "newProjectRoot": "./",
+  "projects": {
+    "charts-angular": {
+      "projectType": "library",
+      "root": "./",
+      "sourceRoot": "src",
+      "prefix": "carbon",
+      "architect": {
+        "build": {
+          "builder": "@angular-devkit/build-angular:ng-packagr",
+          "options": {
+            "project": "ng-package.json"
+          },
+          "configurations": {
+            "production": {
+              "tsConfig": "tsconfig.lib.prod.json"
+            },
+            "development": {
+              "tsConfig": "tsconfig.lib.json"
+            }
+          },
+          "defaultConfiguration": "production"
+        },
+        "test": {
+          "builder": "@angular-devkit/build-angular:karma",
+          "options": {
+            "main": "src/test.ts",
+            "tsConfig": "tsconfig.spec.json",
+            "karmaConfig": "karma.conf.js"
+          }
+        },
+        "storybook": {
+          "builder": "@storybook/angular:start-storybook",
+          "options": {
+            "configDir": ".storybook",
+            "tsConfig": ".storybook/tsconfig.json",
+            "browserTarget": "charts-angular:build",
+            "compodoc": false,
+            "compodocArgs": [
+              "-e",
+              "json",
+              "-d",
+              "./"
+            ],
+            "port": 9006,
+            "quiet": true
+          }
+        },
+        "build-storybook": {
+          "builder": "@storybook/angular:build-storybook",
+          "options": {
+            "configDir": ".storybook",
+            "tsConfig": ".storybook/tsconfig.json",
+            "browserTarget": "charts-angular:build",
+            "compodoc": false,
+            "compodocArgs": [
+              "-e",
+              "json",
+              "-d",
+              "./"
+            ],
+            "outputDir": "demo/bundle",
+            "quiet": true
+          }
+        }
+      }
+    }
+  },
+  "cli": {
+    "analytics": false,
+    "packageManager": "yarn"
+  },
+  "schematics": {
 		"@schematics/angular:component": {
-			"prefix": "ibm",
-			"styleext": "css"
+			"prefix": "ibm"
 		},
 		"@schematics/angular:directive": {
 			"prefix": "ibm"
 		}
 	}
-}
+}
```

Comment:
```suggestion
}

```

Reply:
This is a prettier thing. It doesn't want the final CR or LF at the end of JSON or HTML files.